src/store/logReducer.js

```diff
@@ -13,7 +13,7 @@ const logPending = (state) => ({
 const logFulfilled = (state, { payload }) => ({
   ...state,
   loaded: true,
-  results: payload,
+  ...(payload || {}),
```

Comment:
I think its a bit more readable to initialize the `payload` in the function arguments instead 
```jsx
const logFulfilled = (state, { payload = {} }) => ({
  ...state,
  loaded: true,
  ...payload,
})
```

Also what about the order? Is it possible that the `loaded` value is overriden from the payload?

Reply:
I wouldn't bother by that, payload should contain just the data from API, if it sends down `loaded` value, then something strange is happening.